libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -326,13 +326,17 @@ void AP_Camera::send_feedback(mavlink_channel_t chan) const
         // completely ignore this failure!  AHRS will provide its best guess.
     }
 
-    float altitude, altitude_rel;
-    if (current_loc.relative_alt) {
-        altitude = current_loc.alt+ahrs.get_home().alt;
-        altitude_rel = current_loc.alt;
-    } else {
-        altitude = current_loc.alt;
-        altitude_rel = current_loc.alt - ahrs.get_home().alt;
+    int32_t altitude;
+    if (!current_loc.get_alt_cm(Location::AltFrame::ABSOLUTE, altitude)) {
+        // completely ignore this failure!  this is a shouldn't-happen
```

Comment:
Why the if, if  we ignore the return ?

Reply:
So we can put a comment in there.

There will be a follow-up PR here proposing that we `return false;` from this method in the case we can't get either our location or convert these altitudes (I kind of promised MdB that one).  I have vague recollections that it is important for us to send this feedback message even if we don't have position for some reason - so that future PR wil at least ensure that we get a comment explaining why we must send the message even without position.  I *think* it comes down to, "so people can test their vehicles inside".